test/hotspot/jtreg/compiler/print/TestPrintAssemblyDeoptRace.java

```diff
@@ -23,10 +23,10 @@
 
 /*
  * @test
- * @bug 8258229
+ * @bug 8258229 8356310
```

Comment:
I'm not sure how much it helps a future reader that wants to figure what the test is about. Doesn't hurt so much either tho.

Reply:
That's a good point. Since it's only a follow-up fix to make the test work with product, you don't need to add the bug number.